mne/dipole.py

```diff
@@ -466,12 +466,14 @@ def _read_dipole_text(fname):
 # #############################################################################
 # Fitting
 
-def _dipole_forwards(fwd_data, whitener, rr, n_jobs=1):
+def _dipole_forwards(fwd_data, whitener, rr, n_jobs=1, extra=None):
```

Comment:
Because it is a temporal method, OTP shouldn't negatively affect dipole fitting, whereas for spatial operators like SNS you must apply the SNS operator to the lead field otherwise bias results